pages/search/index.tsx

```diff
@@ -61,7 +61,9 @@ export default function Search({ results }) {
         {totalResults ? (
           <>
             <Heading level="three">
-              {`Displaying 1-50 of ${totalResults.toLocaleString()} results for keyword "${
+              {`Displaying ${
+                totalResults > 50 ? "1-50" : totalResults.toLocaleString()
```

Comment:
This should eventually include logic to show the actual number in case the number of results is less than 50, but that's out of scope for this PR.